crates/deltalake-core/src/kernel/arrow/mod.rs

```diff
@@ -8,12 +8,25 @@ use arrow_schema::{
 };
 use lazy_static::lazy_static;
 
-use super::schema::{ArrayType, DataType, MapType, PrimitiveType, StructField, StructType};
+use super::{
+    schema::{ArrayType, DataType, MapType, PrimitiveType, StructField, StructType},
+    ActionType,
+};
 
 pub mod schemas;
 
-const MAP_KEYS_NAME: &str = "keys";
-const MAP_VALUES_NAME: &str = "values";
+const MAP_ROOT_DEFAULT: &str = "entries";
+const MAP_KEY_DEFAULT: &str = "keys";
+const MAP_VALUE_DEFAULT: &str = "values";
+const LIST_ROOT_DEFAULT: &str = "element";
```

Comment:
Will this be backwards compatible? Old writers wrote `item` instead of element, this may become problematic when a table schema is constructed from the StructType and the ArrowSchema is actually the same except for the nested field name.

Reply:
in the sense that all our tests are passing, yes. while the written parquet files might look different , but this is already the case as different writers handle this differently. (e.g. spark and arrow).

For this PR this change also is not too important, as we do not yet try to enforce this schema speciufically on parquet reads. I have an improved version for parquet reading in preparation where we can go deeper into the discussion...

Reply:
Alright sounds good!